client/js/utils.js

```diff
@@ -98,6 +99,22 @@ function toggleNotificationMarkers(newState) {
 	viewport.toggleClass("notified", newState);
 }
 
+function updateTitle() {
+	let title = $(document.body).data("app-name");
+	const chanTitle = $("#sidebar").find(".chan.active").attr("aria-label");
+
+	if (chanTitle.length > 0) {
+		title = `${chanTitle} — ${title}`;
+	}
+
+	// change title to display event count
+	// for now, we only show the number of "highlight" class badges
+	const alertEventCount = document.getElementsByClassName("badge highlight").length;
```

Comment:
Can you update this to count up `data-highlight`? :)

Also use jQuery selectors.